backend/fms_core/models/sample.py

```diff
@@ -264,65 +127,27 @@ def add_error(field: str, error: str):
 
         self.normalize()
 
-        # Sample Kind
-        sample_kind_choices = (Sample.BIOSPECIMEN_TYPE_NA_CHOICES if self.extracted_from
-                               else Sample.BIOSPECIMEN_TYPE_CHOICES)
-
-        if self.sample_kind_id is not None:
-            if self.sample_kind.name not in frozenset(c[0] for c in sample_kind_choices):
-                add_error(
-                    "sample_kind",
-                    (f"Sample Kind name {self.sample_kind.name} not valid for "
-                     f"{' extracted' if self.extracted_from else ''} sample {self.name}"),
-                )
-            # Check concentration fields given sample_kind
-            if self.sample_kind.name in Sample.BIOSPECIMEN_TYPES_CONC_REQUIRED and self.concentration is None:
-                add_error("concentration", "Concentration must be specified if the sample_kind is DNA")
-
-            # Check tissue source given extracted_from
-
-            if self.tissue_source and self.sample_kind.name not in Sample.BIOSPECIMEN_TYPES_NA:
-                add_error("tissue_source", "Tissue source can only be specified for a nucleic acid sample.")
-
-            if self.transferred_from and self.sample_kind != self.transferred_from.sample_kind:
-                    add_error("sample_kind", "Sample kind need to remain the same during transfer")
+        DerivedSample = apps.get_model("fms_core", "DerivedSample")
```

Comment:
Why use the apps.get_model when you're outside of a migration here?
For now we used to import the other models in an import statement (from fms_core.models import DerivedSample).

Might not cause such a problem, but the apps.get_model looks up directly what is in the db, it does not rely on Django models

Reply:
You mean it may affect performance ... Ok i'll import it ;).